tests/CustomGraphQLServer.integration.test.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import express, {Express} from 'express'
+import {Server} from 'node:http'
+import {
+    GraphQLServer,
+    GraphQLServerOptions,
+    GraphQLServerRequest,
+    GraphQLServerResponse
+} from '../src/'
+import {
+    usersQuery,
+    userSchema,
+    userSchemaResolvers,
+} from './ExampleSchemas'
+import {
+    ExecutionResult,
+} from 'graphql'
+import {
+    fetchResponse,
+    GRAPHQL_SERVER_PORT,
+    INITIAL_GRAPHQL_SERVER_OPTIONS,
+    LOGGER
+} from './TestHelpers'
+
+let customGraphQLServer: GraphQLServer
+let graphQLServer: Server
+
+beforeAll(async() => {
+    graphQLServer = setupGraphQLServer().listen({port: GRAPHQL_SERVER_PORT})
+    console.info(`Starting GraphQL server on port ${GRAPHQL_SERVER_PORT}`)
+})
+
+afterAll(async() => {
+    await graphQLServer.close()
+})
+
+test('Should return value from context instead of user data ', async() => {
+    customGraphQLServer.setOptions({
+        schema: userSchema,
+        rootValue: userSchemaResolvers,
+        logger: LOGGER,
+        debug: true,
+        reassignAggregateError: false,
+        contextFunction: () => {
+            return {
+                'customText': 'customResponse'
+            }
+        }
+    })
+    const response = await fetchResponse(`{"query":"${usersQuery}"}`)
+    const responseObject = await response.json()
+    expect(responseObject).toStrictEqual('customResponse')
+    expect(responseObject.extensions).toBe(undefined)
```

Comment:
Isn't there a `toBeUndefined()` assertion function?

Reply:
Yes, that's available. Changed it